packages/active-campaign-client/.env.example

```
@@ -0,0 +1,3 @@
+AC_BASE_URL=https://uqido1728911682.activehosted.com
```

```suggestion
AC_BASE_URL=your_account_url
```
or something similar